packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -3,6 +3,7 @@
 // found in the LICENSE file.
 
 import 'dart:async';
+import 'dart:io' as io; // ignore: dart_io_import;
```

Comment:
What's the relationship between the `dart_io_import` lint and the forbidden imports test?

Reply:
The implication is that we should general prefer the io wrappers instead of dart:io.

In this case, the signal wrappers don't work though, since the process instances expects an exact type (uses a private field maybe?)

Reply:
I'm asking about the enforcement mechanism. Should we be preferring to ignore the lint, or to add to the allowlist in the test?

Reply:
Ahh, yeah

I actually prefer the ignore because it is localized. It is easy to forget and leave entries in the allowlist